services/web/server/tests/integration/02/test_computation.py

```diff
@@ -44,26 +45,93 @@
 from tenacity.wait import wait_fixed
 from yarl import URL
 
-current_dir = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
-
 API_VERSION = "v0"
 API_PREFIX = "/" + API_VERSION
 
+
 # Selection of core and tool services started in this swarm fixture (integration)
 pytest_simcore_core_services_selection = [
-    "redis",
-    "rabbit",
-    "director",
-    "director-v2",
-    "postgres",
     "dask-scheduler",
     "dask-sidecar",
+    "director-v2",
+    "director",
+    "migration",
+    "postgres",
+    "rabbit",
+    "redis",
     "storage",
 ]
 
 pytest_simcore_ops_services_selection = ["minio", "adminer"]
 
 
+# HELPERS ----------------------------------------------------------------------------
+
+ExpectedResponse = namedtuple(
+    "ExpectedResponse",
+    ["ok", "created", "no_content", "not_found", "forbidden", "locked", "accepted"],
+)

```

Comment:
I tend to replace these namedtuple by dataclass now.
pylance is happier, it seems it is slightly faster.